### PR TITLE
Add `mixHash` to `eth_getBlockByNumber` output

### PR DESCRIPTION
**Pull request by rahul10-pu**
This PR is for this [issue](https://github.com/aurora-is-near/aurora-relayer/issues/54) #54
The main purpose of this pr is to include mixHash enity in the response of eth_getBlockByNumber. 
http post http://localhost:8545 jsonrpc=2.0 id=1 method=eth_getBlockByNumber params:='["73429683"]'
```
{
    "id": "1",
    "jsonrpc": "2.0",
    "result": {
        "difficulty": "0x0",
        "extraData": "0x",
        "gasLimit": "0x0",
        "gasUsed": "0x0",
        "hash": "0x4f547bf73900f58328b6da64e82099ecdcc331062486c228c363e97c48fa82c4",
        "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
        "miner": "0x0000000000000000000000000000000000000000",
        "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
        "nonce": "0x0000000000000000",
        "number": "0x46072b3",
        "parentHash": "0x884b132d54830c13a987b5f182e23e26973e319393bbef5b0f70fd4fb863a885",
        "receiptsRoot": "0x3d94b31b19c4a191ea5e4fd33ad94cba56ebf74dcc6d8564bf89933ce222b59f",
        "sha3Uncles": "0x0000000000000000000000000000000000000000000000000000000000000000",
        "size": "0x8b6d",
        "stateRoot": "0x67c375d37fabc6a28a6a7952b21170d12e922d720173ad379f2d032f1f24cb85",
        "timestamp": "0x61a7275c",
        "totalDifficulty": "0x0",
        "transactions": [],
        "transactionsRoot": "0x50ae6dca7cc0014767708eb28f3e41cae8c5bc37bbad24eba08d9d4923ab9c78",
        "uncles": []
    }
}
```

### CODE REVIEW

**Comment by joshuajbouw** [comment]
In the code it returns empty bytes, but why is the example return not empty bytes?

File: etc/schema/types/block_result.sql [hunk 1]
```diff
@@ -17,5 +17,6 @@ CREATE TYPE block_result AS (
     "size" int4,
     "gasLimit" int8,
     "gasUsed" int8,
+    "mixHash" bytea,         -- 32 bytes
```
Comment (artob) [inline etc/schema/types/block_result.sql:20]: The column type should be `hash` instead of `bytea`, similarly to e.g. `block_result` columns.

**Review by artob — CHANGES_REQUESTED** [review]
LGTM, but needs one correction and also needs instructions to reload the affected stored procedure added to [the changelog](https://github.com/aurora-is-near/aurora-relayer/blob/master/CHANGES.md). Watch out for a merge conflict with #132 which will also update the changelog.

**Review by artob — APPROVED** [review]
LGTM :+1: And I've verified that all the other uses of `RETURNS block_result` defer to this one, so looks good to go.

File: etc/schema/types/block_result.sql (hunk 1 above)
Comment (artob) [reply etc/schema/types/block_result.sql]: @rahul10-pu Please make sure to press the 'Resolve' button on discussions when they've been sorted.

**Pull request merged**